easybuild/easyconfigs/g/GLib/GLib-2.34.3-foss-2015b.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.34.3'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://ftp.gnome.org/pub/gnome/sources/glib/%(version_major_minor)s/']
+sources = ['glib-%(version)s.tar.xz']
+
+dependencies = [
+    ('libffi', '3.0.13'),
+    ('gettext', '0.18.2'),
+    ('libxml2', '2.9.1'),
+]
+builddependencies = [('Python', '2.7.3')]
```

Comment:
Just nervousness that it might break something.

I know you prefer the latest version, generally. Am happy to try that.